SCClassLibrary/Common/GUI/PlusGUI/Control/ServerTreeView.sc

```diff
@@ -152,8 +153,10 @@ ServerTreeView {
 				size
 			};
 			defer {
-				view.bounds = Rect(0, 0, 400, max(400, tabSize * (countSize.value(levels) + 2)));
-				view.refresh;
+				//view.bounds = Rect(0, 0, 400, max(view.bounds.height, tabSize * (countSize.value(levels) + 2)));
+				//view.refresh;
```

Comment:
Is there a reason these commented out lines are kept?

Reply:
Because I am not sure about my fix.... Someone may find a better solution during the review process or in the future.

I can remove it if reviewers find that this fix is good.

Reply:
@telephone
Could you take a closer look at this PR? I think I have fixed the problem you reported and another one I found when checking the group node issue.
Sorry for not checking the group nodes in #6294 and the other issue I explained here.